wolfcrypt/src/ecc.c

```diff
@@ -11242,6 +11258,10 @@ int wc_ecc_encrypt(ecc_key* privKey, ecc_key* pubKey, const byte* msg,
     if (ret != 0)
         return ret;
 
+#ifndef WOLFSSL_ECIES_OLD
+    pubKeySz = wc_ecc_size(privKey) * 2 + 1;
```

Comment:
pubKeySz is word32 and wc_ecc_size is int. Are you checking zero response? I realize the +1, which is the point type. Perhaps we can add an enum /macro for this? Ideally one in ecc.h around `ECC_POINT_UNCOMP`, but in FIPS that doesn't work.

Reply:
Changed to use static function.